modules/landing-pages/assets/js/admin/landing-pages.js

```diff
@@ -2,10 +2,11 @@ import AdminMenuHandler from 'elementor-admin/admin-menu';
 
 export default class LandingPagesHandler extends AdminMenuHandler {
```

Comment:
Why is the file needed?

Reply:
This has specific landing page functionality. the `AdminMenuHandler` class is generic, allowing highlighting and un-highlighting menu items.